data/human/pirate jobs.txt

```diff
@@ -2683,6 +2720,8 @@ mission "Raid on Merchants (South)"
 		government "Merchant"
 		system
 			distance 1 3
+			neighbor
+				not government "Pirate"
```

Comment:
Would it make sense to apply this to the North and Core versions of this job as well?
Currently, it seems like you could take the job on Freedom, and have the targets spawn in Alnilam, or on Stormhold and have targets in Atik, which probably shouldn't be the case.

Reply:
Don’t navy ships patrol there?

Reply:
> Also, why such a large reputation penalty with pirates in the slave jobs? The next highest penalty I see for failing a pirate job is 3.

That would also be a jump from -5 penalty to an atrocity for the republic/syndicate/free worlds, which sets reputation to minimum 0 and then decreases it by 10, however, I think I can probably make it -9 instead.